src/components/interfaces/HMI_API.xml

```diff
@@ -2410,7 +2727,6 @@
   <description>
     Describes different audio type configurations for PerformAudioPassThru.
     e.g. 8kHz,8-bit,PCM
-    The audio is recorded in monaural.
```

Comment:
Should not be removed

Reply:
@jacobkeeler fixed 7959c85f8296c3ded56e7ee8e9271d5b0940ac4d